File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/witnesses/F09.P1/go/c2/w.go
package main

import (
	"fmt"
	"math"
)

func probe() bool {
	s1 := math.NaN()
	s2 := math.NaN()
// BEGIN PROBE F09.P1
eq := s1 == s2
// END PROBE F09.P1
	return eq
}

func main() { fmt.Println(probe()) }

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/kotlin/chunk_B/f09p1_V2_int/w.kt
// F09.P1 kotlin WITNESS V2 (A4 = fixed-width signed integer). Fragment byte-identical.
fun probe(): Boolean {
    val s1 = 2147483647
    val s2 = 2147483647
    // BEGIN PROBE F09.P1
    val eq = s1 == s2
    // END PROBE F09.P1
    return eq
}
fun main() { println("eq=" + probe()) }
